config-templates/redhat/odf-local/changelog.json

```diff
@@ -3,21 +3,21 @@
         "4.17": {
             "status": "supported",
             "latest-revision": {
-                "revision": "3",
-                "release-date": "08 April 2025",
-                "cves": "CVE-2025-24528, CVE-2024-5535, CVE-2020-11023",
-                "golang": "1.23.7",
+                "revision": "4",
+                "release-date": "23 April 2025",
+                "cves": "CVE-2025-22872",
+                "golang": "1.23.11",
```

Comment:
Are u using common golang builder image? I see latest golang image is `1.23.8`
https://github.ibm.com/alchemy-registry/base/releases/tag/v0.1.899
Please confirm.